_search-plugins/ubi/index.md

```diff
@@ -11,39 +11,66 @@ redirect_from:
+* An (optional!) OpenSearch [plugin](https://github.com/opensearch-project/user-behavior-insights) that streamlines the recording of query data.
 
+Advanced features in OpenSearch, such as the Search Quality Evaluation Framework, and the Hybrid Search Optimizer all build on the UBI specification.
 
-*Tutorials and how-to guides*
+<!-- vale off -->
 
-| Link | Description |
-| :--------- | :------- |
-| [UBI plugin](https://github.com/opensearch-project/user-behavior-insights) | How to install and use the UBI plugin. |
-| [UBI client data structures]({{site.url}}{{site.baseurl}}/search-plugins/ubi/data-structures/)  | Sample JavaScript structures for populating the event store. |
-| [Example UBI query DSL queries]({{site.url}}{{site.baseurl}}/search-plugins/ubi/dsl-queries/)  | How to write queries for UBI data in OpenSearch query DSL. |
-| [Example UBI SQL queries]({{site.url}}{{site.baseurl}}/search-plugins/ubi/sql-queries/)  | How to write analytic queries for UBI data in SQL. |
-| [UBI dashboard tutorial]({{site.url}}{{site.baseurl}}/search-plugins/ubi/ubi-dashboard-tutorial/) | How to build a dashboard containing UBI data. |
-| [Chorus Opensearch Edition](https://github.com/o19s/chorus-opensearch-edition/?tab=readme-ov-file#structured-learning-using-chorus-opensearch-edition) katas | A series of structured tutorials that teach you how to use UBI with OpenSearch through a demo e-commerce store. |
+<table>
+    <tr style="vertical-align: top;">
+        <td>
+            <h2>Tutorials</h2>
+            <ul>
+                <li><a href="#">Learn to use <code>ubi.js</code></a></li>
+                <li><a href="#">Using OpenSearch Ingestion from AWS with UBI</a></li>
+                <li><a href="#">Link 3</a></li>
+            </ul>
+        </td>
+        <td>
+            <h2>How To Guides</h2>
+            <ul>
+                <li><a href="https://github.com/opensearch-project/user-behavior-insights">How to install and use the UBI plugin</a><b>do we keep this</b></li>
+                <li><a href="{{site.url}}{{site.baseurl}}/search-plugins/ubi/ubi-dashboard-tutorial/">How to build a custom dashboard with UBI data.</a></li>
+                <li><a href="{{site.url}}{{site.baseurl}}/search-plugins/ubi/dsl-queries/">How to write queries for UBI data in OpenSearch query DSL.</a></li>
+                <li><a href="{{site.url}}{{site.baseurl}}/search-plugins/ubi/sql-queries/">How to write analytic queries for UBI data in SQL.</a></li>
+                <li><a href="https://github.com/o19s/chorus-opensearch-edition/blob/main/katas/006_protecting_sensitive_information.md">How to protect sensistive information when using UBI.</a> <small>Part of Chorus series.</small></li>
+            </ul>
+        </td>
+    </tr>
+    <tr style="vertical-align: top;">
+        <td>
+            <h2>Explanation</h2>
+            <ul>
+                <li><a href="https://UBISearch.dev">Why UBI?</a></li>
```

Comment:
I think this page should answer "Why UBI?". If it doesn't no one will follow any link presented on this page.

Additionally, two links to the community page are not necessary.

Reply:
good call.